vulnfeeds/cves/versions.go

```diff
@@ -218,3 +218,18 @@ func ExtractVersionInfo(cve CVEItem, validVersions []string) (VersionInfo, []str
 	}
 	return v, notes
 }
+
+func CPEs(cve CVEItem) []string {
+	cpesMap := map[string]bool{}
+	for _, node := range cve.Configurations.Nodes {
+		for _, match := range node.CPEMatch {
+			cpesMap[match.CPE23URI] = true
+		}
+	}
+
+	var cpes []string
```

Comment:
you could pre-allocate the size of this slice with `len(cpesMap)`, i think (but avoid allocations vs readability! your choice!)

Reply:
Right, but this isn't exactly performance critical so I think we can avoid this for readability :)